py_neuromodulation/nm_settings.py

```diff
@@ -3,15 +3,15 @@
 
 import pandas as pd
 
-from py_neuromodulation import nm_IO
+from py_neuromodulation.nm_IO import PYNM_DIR, read_settings
```

Comment:
Do you think it could make sense to add PYNM_DIR to the `__init__.py` file? Intuitively I would have guessed it would belong there

Reply:
Probably yes. It was mostly used in nm_IO so I defined it first there, then I found it used in other places. I already moved it there.